urbandict-search/Dockerfile

```
@@ -0,0 +1,16 @@
+FROM pytorch/pytorch:latest
+
+WORKDIR /
+
+ENV DATA_DIR /
+
+COPY . /
+
+RUN python -m pip install --no-cache-dir --upgrade pip && \
+    pip install --no-cache-dir -r requirements_docker.txt
+
+RUN python -c "from transformers import DistilBertModel, DistilBertTokenizer; x='distilbert-base-cased'; DistilBertModel.from_pretrained(x); DistilBertTokenizer.from_pretrained(x)"
+
+RUN python prepare_data.py && python app.py -t index -n 10000
+
```

After building the index, delete the data file in order to keep the docker small.

```suggestion
RUN rm ${DATA_PATH}
```